README.md

```diff
@@ -117,52 +117,91 @@ There are several demonstrations of common cases.
 ```
 
 #### 3. Deploy and run karmada control plane:
+
+Choose a way:
+
+##### 3.1. I have not any cluster
```

Comment:
nit: It would be helpful adding a link after here, to quickly jump to `3.2. I have present cluster for installing`

Reply:
done